NPPESAPI.net/NPPESAPI.net.csproj

```diff
@@ -11,12 +11,16 @@
     <LangVersion>latest</LangVersion>
     <RepositoryUrl>https://github.com/ForcuraCo/npi-api-net</RepositoryUrl>
     <PackageLicenseUrl>https://www.apache.org/licenses/LICENSE-2.0</PackageLicenseUrl>
-    <Authors>ghost1face,Forcura</Authors>
+    <Authors>ghost1face,Forcura,Reality Engineering</Authors>
```

Comment:
You are not an author of the package/library, but based on your contributions will be recognized as a contributor to this repository.  Please remove.

Reply:
Understood. Change has been made.